doc/specs/vulkan/chapters/VK_EXT_debug_report.txt

```diff
@@ -124,11 +128,15 @@ include::../api/funcpointers/PFN_vkDebugReportCallbackEXT.txt[]
   * pname:pMessage is a null-terminated string detailing the trigger
```

Comment:
Should perhaps also say pname:pLayerPrefix is a null-terminated string (i.e. it cannot be e.g. `nullptr` I assume)